data/112/578/917/9/1125789179.geojson

```diff
@@ -18,6 +18,7 @@
     "lbl:min_zoom":13.5,
     "mz:hierarchy_label":1,
     "mz:is_current":1,
+    "mz:is_funky":1,
```

Comment:
Why are we marking this one current and funky?

Reply:
https://github.com/whosonfirst-data/whosonfirst-data-admin-ee/pull/11#discussion_r463334210, same applies here